va/va.c

```diff
@@ -721,6 +721,10 @@ static VAStatus va_new_opendriver(VADisplay dpy)
         }
 
         vaStatus = va_openDriver(dpy, drivers[i]);
+        /* Try as path as fallback */
+        if (vaStatus != VA_STATUS_SUCCESS)
```

Comment:
Copying from PR description:
> va: When driver loading fails with the provided name by the backend, try full path loading as fallback
> This fixes loading of the registry driver full path for a given device
> This fixes loading LIBVA_DRIVER_NAME pointing to an absolute path which we want to support.
> No changes to existing loading mechanisms

This needs to be added to the commit message of 2nd commit (maybe paraphrased). Main thing to highlight is that LIBVA_DRIVER_NAME now allows full path specification.